docs/vectorizer.md

```diff
@@ -146,7 +146,7 @@ To find the closest embeddings to a query, use this canonical SQL query:
 ```sql
 SELECT 
    chunk,
-   embedding <=> ai.openai_embed(...) as distance
+   embedding <=> ai.openai_embed('text-embedding-3-small', <query>) as distance
```

Comment:
This also needs `dimensions => 768` as a parameter right? I remember copying this and it not working otherwise.

Reply:
Good idea. Can I leave this to you in a followup pr?